core/src/test/java/io/questdb/test/griffin/ExplainPlanTest.java

```diff
@@ -1701,117 +1698,107 @@ public void testFilterOnExcludedIndexedSymbolManyValues() throws Exception {
             );
 
             assertPlan("select s, count() from trips where s is not null",
-                    "GroupBy vectorized: false\n" +
+                    "Async Group By workers: 1\n" +
```

Comment:
Is there any benefit of Async Group By with 1 worker? Or 1 is it purely artificial and cannot happen outside of the test code?

Reply:
You'll see `Async Group By workers: 1` if your shared worker pool has only one thread. It's the same thing as with parallel filtering: we don't bother disabling the filtering when there is only a single shared worker thread.